PhpAmqpLib/Message/AMQPMessage.php

```diff
@@ -56,5 +53,7 @@ public function __construct($body = '', $properties = null)
     public function setBody($body)
     {
         $this->body = $body;
+
+        return $this;
```

Comment:
Cool. I think it's fine to have it, just that the parent class doesn't implement this kind of API